examples/README.md

```diff
@@ -23,6 +23,36 @@ send back a launch_context containing, among other things, an access token to
 set in the Authorization header and use for subsequent FHIR requests (to the
 ISS).
 
+## [examples/public-smart-ehr](./public-smart-ehr)
+
+The public-smart-ehr example is almost identical to the confidential-smart-ehr code above, but it is assumed that this app would be downloaded to a device and run in an environment that cannot protect a client secret (eg. a browser). This means that the public-smart-ehr app:
+
+  - is assumed to run on an end-user's device rather than on a trusted server
+  - cannot protect secret variables (in this case, CLIENT_SECRET)
+  - should be hosted within a trusted server environment
+
+The app's launch URL is required to be preregistered with the EHR. This required step helps to mitigate security risks.
+Because the public app is the less secure option with no ability to store confidential information, it is recommended that refresh tokens for public apps have a shorter lifetime.   
+
+The example provides two routes, `/launch` and `/callback`, through which an EHR
+may launch the SMART app within the EHR's provided launch context.
+
+To use:
```

Comment:
Let's just refer to the instructions above as is done for the third example rather than repeating them here.